client-spark/spark3/src/main/java/org/apache/spark/shuffle/writer/RssShuffleWriter.java

```diff
@@ -240,6 +243,17 @@ protected void checkBlockSendResult(Set<Long> blockIds) throws RuntimeException
     }
   }
 
+  private void fastFailOnceBlockFailed() {
```

Comment:
Could we have a better name? How about `checkIfBlocksFailed()`?

Reply:
Got it

Reply:
Done